protocol/contracts/lease/src/contract/state/dex.rs

```diff
@@ -26,22 +26,15 @@ impl<H> State<H> {
     }
 }
 
-#[cfg(feature = "migration")]
-impl<H> State<H> {
-    pub fn map<MapFn, HTo>(self, map_fn: MapFn) -> State<HTo>
-    where
-        MapFn: FnOnce(H) -> HTo,
-    {
-        State::new(map_fn(self.handler))
-    }
-
-    pub fn inspect<InspectFn, R>(&self, inspect_fn: InspectFn) -> R
-    where
-        InspectFn: FnOnce(&H) -> R,
-    {
-        inspect_fn(&self.handler)
-    }
-}
+// #[cfg(feature = "migration")]
+// impl<H> State<H> {
+//     pub fn map<MapFn, HTo>(self, map_fn: MapFn) -> State<HTo>
+//     where
+//         MapFn: FnOnce(H) -> HTo,
+//     {
+//         State::new(map_fn(self.handler))
+//     }
+// }
```

Comment:
This one was one of your commits, if I am not wrong. Do we really want to keep it commented? AFAIR, that was considered a code smell.

Reply:
right, it is. I did it with the intention to uncomment right in the next commit. It's under review ... and will go in the repo soon.

Thanks for catching it